### PR TITLE
feat(appendFlipped): add flipped version of append

### LINKED ISSUE #333: appendFlipped
One thing I do a lot is `flip(append)`.  I know there is a `concatRight` so how about an `appendRight`? 

```
const appendRight = flip(append);
```

Useful with transformers too which expect (acc, value).

### PR DESCRIPTION
**Pull request by Undistraction**
Adds a version of Ramda's append with the argument order flipped.

closes #333

### CODE REVIEW

**Comment by char0n** [comment]
Will review this tomorrow morning. Thanks

File: src/appendFlipped.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * @func appendFlipped
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.5.0|v2.5.0}
+ * @category List
+ * @sig [a] -> a -> [a]
+ * @param {Array} list The list of elements to add a new item to.
+ * @param {*} el The element to add to the end of the new list.
+ * @return {Array} A new list containing the elements of the old list followed by `el`.
+ * @see {@link http://ramdajs.com/docs/#append|append}
+ * @example
```
Comment (char0n) [inline src/appendFlipped.js]: Drop the padding.
```js
@example

R.append(['write', 'more'], 'tests'); //=> ['write', 'more', 'tests']
R.append([], 'tests'); //=> ['tests']
R.append(['write', 'more'], ['tests']); //=> ['write', 'more', ['tests']]
```

File: src/appendFlipped.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { append, flip } from 'ramda';
+
+
+/**
+ * Returns a new list containing the contents of the given list, followed by the given element.
+ * Ramda's `append`, but with argument order reversed: list then value.
```
Comment (char0n) [inline src/appendFlipped.js]:
```js
Returns a new list containing the contents of the given list, followed by the given element.
Like {@link http://ramdajs.com/docs/#append|R.append} but with arguments order reversed.
```
Would drop the `list then value` part. It's apprarent from the signature and jsdoc @params.

File: src/index.d.ts [hunk 3]
```diff
@@ -790,6 +790,12 @@ declare namespace RamdaAdjunct {
          */
         compact(list: Array<any>): Array<any>;
 
+        /**
+         * Returns a new list containing the contents of the given list, followed by the given
```
Comment (char0n) [inline src/index.d.ts]:
```ts
Returns a new list containing the contents of the given list, followed by the given element.
Like {@link http://ramdajs.com/docs/#append|R.append} but with arguments order reversed.
```

**Comment by Undistraction** [comment]
@char0n That's done. It's not picking up the added test, but it is there.

**Comment by char0n** [comment]
Looks good. Nice work !

**Pull request merged**